Packs/CommonScripts/Scripts/FormatURL/FormatURL.py

```diff
@@ -193,6 +216,7 @@ def format_single_url(non_formatted_url: str) -> List[str]:
     # Common handling for unescape and normalizing
     non_formatted_url = unquote(unescape(non_formatted_url.replace('[.]', '.')))
     formatted_url = replace_protocol(non_formatted_url)
+    formatted_url = remove_special_chars_from_start_and_end_of_url(formatted_url)
     formatted_url = remove_brackets_from_end_of_url(formatted_url)
```

Comment:
Do we still need `remove_brackets_from_end_of_url`?

Reply:
I think so, "remove_special_chars_from_start_and_end_of_url" only handles the scenario where there is the same special char at the beginning and end of URL, for example: [url], (url) etc.
I guess we still want to remove single brackets from the end of the URL, no? @Ni-Knight